packages/frontend/src/components/atoms/input.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+
+import { cn } from '@/lib/utils';
+
+export interface InputProps
+  extends React.InputHTMLAttributes<HTMLInputElement> {}
+
+const Input = React.forwardRef<HTMLInputElement, InputProps>(
+  ({ className, type, ...props }, ref) => {
+    return (
+      <input
+        type={type}
+        className={cn(
+          'flex h-10 focus:text-red w-full px-2 text-sm ring-offset-background file:border-0 file:bg-transparent file:text-sm file:font-medium focus-visible:outline-none  disabled:cursor-not-allowed disabled:opacity-50',
+          className
+        )}
+        ref={ref}
+        placeholder="Email address"
```

Comment:
this is a generic input component, we can't assume that the placeholder will be an "Email Address" 
the place holder should be extracted from the props, and by default it should be empty string